File: Modul 4/Soal 3/PRAK403-2410817220018-ANGGRAENI-DWI-ZAHRA.c
#include <stdio.h>

int main() {
    int angka1, angka2;

    printf("Masukkan Angka Pertama: ");
    scanf("%d", &angka1);
    printf("Masukkan Angka Kedua: ");
    scanf("%d", &angka2);


    if (angka1 > angka2) {
        for (int i = 0; i <= (angka1 - angka2); i++) {
            printf("%d %d", angka1 - i, angka2 + i);
            if (i < (angka1 - angka2)) {
                printf(" - "); 
            }
        }
    } else { 
        for (int i = 0; i <= (angka2 - angka1); i++) {
            printf("%d %d", angka1 + i, angka2 - i);
            if (i < (angka2 - angka1)) {
                printf(" - "); 
            }
        }
    }
    printf("\n"); 

    return 0;
}

File: Modul 4/Soal 3/PRAK403-2410817220018-ANGGRAENI DWI ZAHRA.py
angka1 = int(input("Masukkan Angka Pertama: "))
angka2 = int(input("Masukkan Angka Kedua: "))

if angka1 < angka2:
    for i in range(angka1, angka2 + 1):
        print(i, angka2 - (i - angka1), end=" - ")
else:
    for i in range(angka2, angka1 + 1):
        print(angka1 - (i - angka2), i, end=" - ")

print("\b\b  ")  